bool recalculate
---
geometry_msgs/PoseStamped table_centroid